src/prop/prop_engine.cpp

```diff
@@ -453,6 +441,39 @@ Node PropEngine::getPreprocessedTerm(TNode n)
   return tpn.isNull() ? Node(n) : tpn.getNode();
 }
 
+Node PropEngine::getPreprocessedTerm(TNode n,
+                                     std::vector<Node>& skAsserts,
+                                     std::vector<Node>& sks)
+{
+  // get the preprocessed form of the term
+  Node pn = getPreprocessedTerm(n);
+  // initialize the set of skolems and assertions to process
+  std::vector<theory::TrustNode> toProcessAsserts;
+  std::vector<Node> toProcess;
+  d_theoryProxy->getSkolems(pn, toProcessAsserts, toProcess);
+  size_t index = 0;
+  // until fixed point is reached
+  while (index < toProcess.size())
+  {
+    theory::TrustNode ka = toProcessAsserts[index];
+    Node k = toProcess[index];
+    index++;
```

Comment:
```suggestion
    Node k = toProcess[index++];
```

Reply:
I prefer 2 separate lines, it's confusing to me whether the index is incremented before or after the index lookup.